superset-frontend/plugins/plugin-chart-period-over-period-kpi/src/PopKPI.tsx

```diff
@@ -30,16 +34,51 @@ const ComparisonValue = styled.div<PopKPIComparisonValueStyleProps>`
   `}
 `;
 
+const SymbolWrapper = styled.div<PopKPIComparisonSymbolStyleProps>`
+  ${({ theme, percentDifferenceNumber, comparisonColorEnabled }) => {
+    const defaultBackgroundColor = theme.colors.grayscale.light4;
```

Comment:
Interesting approach with putting js inside of styled component like that! But I wonder if deducing colours from more "high-level" data belongs to a styled component. Maybe we should move that logic to `PopKPI` and just pass `backgroundColor` and `textColor` to `SymbolWrapper`?
But if you think this solution is better then I won't argue, I don't have a strong opinion here

Reply:
hehe yeah, I had it like that before and moved to this, it doesn't feel quite "well" but now that you noticed too, let's move it there instead. Thanks